### ISSUE TITLE
Allow globs for Ansible's role_paths

### ISSUE THREAD

**Issue by elasticdog** [hashicorp/packer#1000]
With the current implementation of the Ansible provisioner, you can specify `role_paths` as an array of strings, but you have to be explicit with every role that you want to upload to the remote machine. This becomes a hassle when you have multiple roles or start using role dependencies, as you have to explicitly include all dependencies, dependencies of dependencies, and so on down the line. This makes things very fragile.

I'd love to have the ability to say:

``` json
  "provisioners": [
    {
      "type": "ansible-local",
      "playbook_file": "ansible/playbooks/packer_jenkins_master.yml",
      "role_paths": [
        "ansible/roles/*"
      ]
    }
  ],
```

...instead of:

``` json
  "provisioners": [
    {
      "type": "ansible-local",
      "playbook_file": "ansible/playbooks/packer_jenkins_master.yml",
      "role_paths": [
        "ansible/roles/common",
        "ansible/roles/git",
        "ansible/roles/java",
        "ansible/roles/jenkins",
        "ansible/roles/nginx"
      ]
    }
  ],
```

**Comment by chilicat** [comment]
From the comments it is not clear if globs is implemented or the feature request is rejected

**Comment by marksteward** [comment]
The issue should probably be renamed. The issue was resolved with #1150 (see above):
> Created a pull request to enable uploading of the whole playbook directory including roles, variables etc.
